diff --git a/buildMobileVLCKit.sh b/buildMobileVLCKit.sh
--- buildMobileVLCKit.sh
+++ buildMobileVLCKit.sh
@@ -109,9 +109,9 @@
          x)
              MACOS=yes
              IOS=no
              BITCODE=no
-             SDK_VERSION=`xcrun --sdk macosx --show-sdk-version`
+             SDK_VERSION=10.15
              SDK_MIN=10.9
              OSVERSIONMINCFLAG=mmacosx
              OSVERSIONMINLDFLAG=macosx
              BUILD_DEVICE=yes
@@ -289,9 +289,10 @@
     python3Path=""
 fi
 
 VLCROOT=${ROOT_DIR}/libvlc/vlc
-export PATH="$python3Path:${VLCROOT}/extras/tools/build/bin:${VLCROOT}/contrib/${HOST_TRIPLET}/bin:${VLC_PATH}:/usr/bin:/bin:/usr/sbin:/sbin"
+[ -f /usr/local/opt/python/bin/python ] || ln -s /usr/local/opt/python/bin/python3 /usr/local/opt/python/bin/python
+export PATH="$python3Path:${VLCROOT}/extras/tools/build/bin:${VLCROOT}/contrib/${HOST_TRIPLET}/bin:${VLC_PATH}:/usr/local/opt/python/bin:/usr/bin:/bin:/usr/sbin:/sbin"
 BUILD_ARCH=`get_buildsystem_arch $BUILD_ARCH`
 
 info "Preparing build dirs"
 
@@ -519,9 +520,9 @@
         --disable-kate \
         --disable-caca \
         --disable-gettext \
         --disable-mpcdec \
-        --enable-upnp \
+        --disable-upnp \
         --disable-gme \
         --disable-srt \
         --disable-tremor \
         --enable-vorbis \
@@ -540,17 +541,18 @@
         --enable-ass \
         --disable-fontconfig \
         --disable-gpg-error \
         --disable-vncclient \
-        --disable-gnutls \
+        --disable-securetransport \
+        --enable-gnutls \
         --disable-lua \
         --disable-luac \
-        --disable-aribb24 \
-        --disable-aribb25 \
-        --enable-vpx \
-        --enable-libdsm \
-        --enable-smb2 \
-        --enable-libplacebo \
+        --enable-aribb24 \
+        --enable-aribb25 \
+        --disable-vpx \
+        --disable-libdsm \
+        --disable-smb2 \
+        --disable-libplacebo \
         --disable-sparkle \
         --disable-growl \
         --disable-breakpad \
         --disable-ncurses \
